cmd/txtar-savedir/script_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/rogpeppe/go-internal/gotooltest"
+	"github.com/rogpeppe/go-internal/testscript"
+)
+
+func TestMain(m *testing.M) {
+	os.Exit(testscript.RunMain(m, map[string]func() int{
+		"txtar-savedir": mainint,
+	}))
+}
+
+func TestScripts(t *testing.T) {
+	p := testscript.Params{Dir: "testdata"}
+	if err := gotooltest.Setup(&p); err != nil {
```

Comment:
As txtar-savedir doesn't use the go command, I think this can be left out.

Reply:
Done